src/core/data/sframe/gl_sframe.cpp

```diff
@@ -923,13 +923,13 @@ gl_sframe gl_sframe::sort(const std::vector<std::pair<std::string, bool>>& colum
 }
 gl_sframe gl_sframe::dropna(const std::vector<std::string>& columns,
                             std::string how) const {
```

Comment:
This API is meant to mimic the python one, so let's go ahead and add in the parameter here as well to mimic the python API, with a default of false, along with a test around line 276 of test/unity/gl_sframe.cxx. After that, this PR should be ready to go.

Reply:
Thanks for the catch!